File: POE2_Launcher.cs
using System;
using System.Diagnostics;

class POE2Launcher
{
    static void Main()
    {
        string url = "https://pathofexile2.game.daum.net/main";

        try
        {
            // Try to launch with Edge
            Process.Start(new ProcessStartInfo
            {
                FileName = "msedge",
                Arguments = url,
                UseShellExecute = true
            });
        }
        catch
        {
            // Fallback: use microsoft-edge protocol
            try
            {
                Process.Start(new ProcessStartInfo
                {
                    FileName = "microsoft-edge:" + url,
                    UseShellExecute = true
                });
            }
            catch
            {
                // Last resort: use default browser
                Process.Start(new ProcessStartInfo
                {
                    FileName = url,
                    UseShellExecute = true
                });
            }
        }
    }
}

File: poe2_launcher.py
import subprocess
import sys
import os

# URL to open
URL = "https://pathofexile2.game.daum.net/main"

# Edge paths to try
EDGE_PATHS = [
    r"C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe",
    r"C:\Program Files\Microsoft\Edge\Application\msedge.exe"
]

def main():
    # Try to find and launch Edge
    for edge_path in EDGE_PATHS:
        if os.path.exists(edge_path):
            subprocess.Popen([edge_path, URL])
            sys.exit(0)

    # Fallback: use microsoft-edge protocol
    subprocess.Popen(["cmd", "/c", "start", "microsoft-edge:" + URL])

if __name__ == "__main__":
    main()
